example_values.yaml

```diff
@@ -1,28 +1,50 @@
-# $CHART_UID & $CHART_GID need to be replaced with actual values
-# PROJECT=$(oc project -q)
-#
-# CHART_UID=$(oc get project ${PROJECT} -o jsonpath="{['metadata.annotations.openshift\.io/sa\.scc\.uid-range']}" | sed "s@/.*@@")
-# CHART_GID=$(oc get project ${PROJECT} -o jsonpath="{['metadata.annotations.openshift\.io/sa\.scc\.supplemental-groups']}" | sed "s@/.*@@")
 
-uid: $CHART_UID
-gid: $CHART_GID
+defaultAirflowRepository: apache/airflow
+defaultAirflowTag: "2.7.0"
+
+executor: KubernetesExecutor
+fernetKey: "7T512UXSSmBOkpWimFHIVb8jK6lfmSAvx4mO6Arehnc="
+webserverSecretKey: "ShouldNotBeInProduction"
+
+scheduler:
+  replicas: 2
+
+webserver:
+  replicas: 2
 
 statsd:
-  securityContext:
-    runAsUser: $CHART_UID
+  uid: null
+
+pgbouncer:
+  uid: null
 
 redis:
-  securityContext:
-    runAsUser: $CHART_UID
+  uid: null
 
 dags:
+  persistence:
+    enabled: false
   gitSync:
-    repo: https://github.com/redhat-na-ssa/airflow-on-openshift.git
-    branch: main
-    subPath: example_dag
+    uid: null
+    enabled: true
+    # repo: https://github.com/redhat-na-ssa/airflow-on-openshift.git
+    # branch: main
+    # subPath: example_dag
 
-postgresql:
-  securityContext:
-    enabled: false
-  containerSecurityContext:
+logs:
+  persistence:
     enabled: false
+
+flower:
+  enabled: false
+
+# ingress:
+#   web:
+#     enabled: true
+
+postgresql:
+  primary:
```

Comment:
I don't see postgresql.primary anywhere in values.yaml of Helm Charts 1.10.0 / Airflow 2.6.2.

Also, what is the production value of disabling podSecurityContext and containerSecurityContext? That is not good practice at all, even in non-prod environments.

Better write clearly that postgres as it comes in docker image form with stable/airflow is not openshift compatible and describe viable alternatives. For example, one could either install a simple deployment using official Red Hat Image for Postgres 13 and use external postgres, or reference postgres installed via Crunchy operator.

Reply:
@shalberd  
1. postgresql is using another helm chart for deployment the bitnami chart. You have to go lookup the values since it is not in the apache repo. It is referenced
2. It's not disabling `podSecurityContext`. OpenShift handles this automatically. We avoid hard coding the values that would be for a k8s install.

Reply:
> Better write clearly that postgres as it comes in docker image form with stable/airflow is not openshift compatible and describe viable alternatives. For example, one could either install a simple deployment using official Red Hat Image for Postgres 13 and use external postgres, or reference postgres installed via Crunchy operator.

Using a ubi based postgres instance would be a good Issue to open for this repo. I can thing of serval ways to address that.

Reply:
There is a solution for postgres hidden in the script, but unused at the moment

See 

https://github.com/opendatahub-io-contrib/airflow-on-openshift/blob/bc3eace4f7cb093c059b73533e2c7c213f99ca3d/scripts/easy_install.sh#L27-L40

Lets take this discussion to a new issue. Feel free to open it @shalberd